.prettierrc.js

```diff
@@ -0,0 +1,9 @@
+/** @type {import('prettier').Config} */
+module.exports = {
+  plugins: ["prettier-plugin-solidity"],
+  printWidth: 120,
+  semi: true,
+  tabWidth: 2,
+  useTabs: false,
+  bracketSpacing: true,
+};
```

Comment:
moved this to a `.js` file so other packages can include/extend in the future with

```
/** @type {import('prettier').Config} */
module.exports = {
  ...require('../../.prettierrc.js'),
  // extended config values or overrides here
};
```